dist/tools/coccinelle/include/riot-standard.h

```diff
@@ -0,0 +1 @@
+#define LL_SEARCH_SCALAR(pkt, tcp, type, GNRC_NETTYPE_TCP) tcp = XXX
```

Comment:
Coccinelle by default ignores macros. In this case, there was a false positive for one of the coccinelle scripts, as coccinelle didn't know that ```LL_SEARCH_SCALAR``` might change ```tcp```.